app/models/book_suggestion.rb

```diff
@@ -0,0 +1,3 @@
+class BookSuggestion < ApplicationRecord
+  belongs_to :user
```

Comment:
Don't forget to add the inverse relation. This is, the User model should have a has_many book_suggestions